builtin/providers/openstack/provider.go

```diff
@@ -105,6 +105,11 @@ func Provider() terraform.ResourceProvider {
 			"openstack_lb_monitor_v1":                  resourceLBMonitorV1(),
 			"openstack_lb_pool_v1":                     resourceLBPoolV1(),
 			"openstack_lb_vip_v1":                      resourceLBVipV1(),
+			"openstack_lbaas_loadbalancer_v2":          resourceLoadBalancerV2(),
```

Comment:
I have no strong feeling either way.  Unless someone else chimes in before Monday, I'll push an update for this.